clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -1150,8 +1150,8 @@ public void testRefreshOffsetLoadInProgress() {
 
         subscriptions.assignFromUser(singleton(tp));
         subscriptions.needRefreshCommits();
-        client.prepareResponse(offsetFetchResponse(tp, Errors.GROUP_LOAD_IN_PROGRESS.code(), "", 100L));
-        client.prepareResponse(offsetFetchResponse(tp, Errors.NONE.code(), "", 100L));
+        client.prepareResponse(offsetFetchResponse(tp, Errors.NONE.code(), "", 100L, Errors.GROUP_LOAD_IN_PROGRESS.code()));
```

Comment:
I'll update the expected responses as you suggested.

Regarding supporting both versions it seems that work would conflict with what is being implemented for KIP-97. Not sure what's the best way to handle it, wait for that to merge first, or move forward with this as is (assuming the latest API version), and then update as part of or after KIP-97.